### PR TITLE
docs: use same expression for rest api

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

File: docs-cn/14-reference/03-connector/java.mdx [hunk 1]
```diff
@@ -805,11 +800,11 @@ Query OK, 1 row(s) in set (0.000141s)
...
-| taos-jdbcdriver 版本 |           主要变化            |
-| :------------------: | :---------------------------: |
-|        2.0.36        |  增加对 schemaless 写入支持   |
-|        2.0.37        |     增加对 json tag 支持      |
-|        2.0.38        | 增加对 RESTful 中批量拉取功能 |
+| taos-jdbcdriver 版本 |            主要变化            |
+| :------------------: | :----------------------------: |
+|        2.0.36        |   增加对 schemaless 写入支持   |
+|        2.0.37        |      增加对 json tag 支持      |
+|        2.0.38        | JDBC REST 连接增加批量拉取功能 |
```
Comment (dingbo8128) [inline docs-cn/14-reference/03-connector/java.mdx:808]: I prefer descending order

**Review by dingbo8128 — APPROVED** [review]

File: docs-cn/14-reference/03-connector/java.mdx [hunk 2]
```diff
@@ -357,7 +352,7 @@ JDBC 连接器可能报错的错误码包括 3 种：JDBC driver 本身的报错
 
 **注意**：
 
-- JDBC-RESTful 实现并不提供参数绑定这种使用方式
+- JDBC REST 连接并不提供参数绑定这种使用方式
```
Comment (sangshuduo) [inline docs-cn/14-reference/03-connector/java.mdx]: 并不提供 => 并不支持，更好一点

File: docs-cn/14-reference/03-connector/java.mdx [hunk 3]
```diff
@@ -627,7 +622,7 @@ public void setNString(int columnIndex, ArrayList<String> list, int size) throws
 
 **注意**：
 
-- JDBC-RESTful 实现并不提供无模式写入这种使用方式
+- JDBC REST 连接并不提供无模式写入这种使用方式
```
Comment (sangshuduo) [inline docs-cn/14-reference/03-connector/java.mdx]: 并不提供 => 暂时不提供，更好一点

File: docs-cn/14-reference/03-connector/java.mdx (hunk 1 above)
Comment (sangshuduo) [reply docs-cn/14-reference/03-connector/java.mdx:808]: 倒序更好

**Review by sangshuduo — CHANGES_REQUESTED** [review]
_preparition still be referred by other connector

**Pull request merged**